### ISSUE TITLE
Menu position not optimal: Title Case should appear in the same group as Upper Case and Lower Case

### ISSUE THREAD

**Issue by alexgleason** [max/atom-title-case#6]
In #4 I said that `Edit > Text` is the best place, but currently its all the way at the bottom of the menu, isolated from its friends `Upper Case` and `Lower Case`.

![screen shot 2015-07-08 at 10 46 30 am](https://cloud.githubusercontent.com/assets/3639540/8574889/a94717a6-255e-11e5-8c18-20364d41ae53.png)

Ideally, `Title Case` would be placed directly after `Lower Case`, followed by the separator. I don't know if this is possible with Atom's API though. But something to look in to.

**Comment by max** [comment]
I don't think this is possible at this time. There is [a discussion](https://discuss.atom.io/t/is-there-a-way-to-specify-menu-item-order-when-inserting-menu-items/14937/14) going on for Electron but I don't think the APIs have made their way into Atom yet.

**Comment by alexgleason** [comment]
Quick update, there is an upstream bug atom/atom#6270 that hasn't seen activity since October 2015. We are still blocked by this.
